components/search_bar/search_bar.tsx

```diff
@@ -7,8 +7,9 @@ import {FormattedMessage, useIntl} from 'react-intl';
 
 import Constants from 'utils/constants';
 import * as Utils from 'utils/utils';
+
+import SuggestionDate from 'components/suggestion/suggestion_date';
 import SearchSuggestionList from 'components/suggestion/search_suggestion_list.jsx';
-import SuggestionDate from 'components/suggestion/suggestion_date.jsx';
 import SuggestionBox from 'components/suggestion/suggestion_box';
```

Comment:
nit : move back in import is same position

Reply:
okay